docs/_static/vdr/v1.yaml

```diff
@@ -150,6 +150,89 @@ paths:
             text/plain:
               schema:
                 type: string
+  /internal/vdr/v1/did/{did}/verificationmethod:
+    parameters:
+      - name: did
+        in: path
+        description: URL encoded DID.
+        required: true
+        example:
+          - "did:nuts:1234"
+        schema:
+          type: string
+    post:
+      summary: Creates a new verificationMethod and adds it to the DID document
+      operationId: addNewVerificationMethod
+      tags:
+        - DID
+      responses:
+        "201":
+          description: "New verification method has been created and added successfully. Returns the DID document."
+          content:
+            application/json+did-document:
+              schema:
+                $ref: '#/components/schemas/VerificationMethod'
+        "403":
```

Comment:
a 403 would indicate that credentials were wrong when trying to access the server. This is more generic so a 400?

Reply:
As can read here: https://tools.ietf.org/html/rfc7231#section-6.5.3 
> a request might be forbidden for reasons unrelated to the credentials.

Which is here the case: your node does not have the rights to change the document. I'm in favour of the more specific 403 against the more general 400 status.